### PR TITLE
Docs/dispensation single signer

### PR DESCRIPTION
**Pull request by Jedi2002**

### CODE REVIEW

**Review by timlind — APPROVED** [review]

File: x/dispensation/Readme-Claim.md [hunk 1]
```diff
@@ -4,17 +4,18 @@
...
 - A claim is deleted once the user is paid out.
-- The claim_create is an auxiliary functionality for the offline calculation api created to calculate user rewards.
+- The `create claim` is the on-chain record for the off-chain components to calculate ValidatorSubsidy and LiquidityMining rewards
 
 ## General use case 
 This is the general use case , the actual implementation might vary a bit
 Any day of the week
  - create claims through this api (on chain)
+Off-Chain components
+ - Create claim snapshots every 200 hours .
```
Comment (marshsif) [inline x/dispensation/Readme-Claim.md:14]: 200 minutes, not hours

File: x/dispensation/Readme-Claim.md [hunk 2]
```diff
@@ -4,17 +4,18 @@
 - The module allows a user to create a Claim .
 - A claim is record , stating that the user want to collect their rewards at the end of the week.
 - A claim is deleted once the user is paid out.
-- The claim_create is an auxiliary functionality for the offline calculation api created to calculate user rewards.
+- The `create claim` is the on-chain record for the off-chain components to calculate ValidatorSubsidy and LiquidityMining rewards
 
 ## General use case 
 This is the general use case , the actual implementation might vary a bit
 Any day of the week
  - create claims through this api (on chain)
```
Comment (marshsif) [inline x/dispensation/Readme-Claim.md:12]: "- community members can submit claims"...

File: x/dispensation/Readme-Claim.md [hunk 3]
```diff
@@ -4,17 +4,18 @@
...
+Off-Chain components
+ - Create claim snapshots every 200 hours .  
+  `@Niko To add the steps here`
 On friday
```
Comment (marshsif) [inline x/dispensation/Readme-Claim.md:16]: Typically on Fridays

File: x/dispensation/Readme-Distrubute.md [hunk 4]
```diff
@@ -16,17 +14,19 @@
...
-    DistributionType DistributionType `json:"distribution_type"`
-    DistributionName string           `json:"distribution_name"`
+	DistributionType DistributionType `json:"distribution_type"`
+	DistributionName string           `json:"distribution_name"`
+	Runner           sdk.AccAddress   `json:"runner"`
 }
 ```
-This is stored in the keeper with the key DistributionType_DistributionName for historical records. Therefore, the combination of type and name needs to be unique
+This is stored in the keeper with the key DistributionName_DistributionType for historical records. 
+The Distribution name is BlockHeight_Distributor ,Therefore, the combination BlockHeight_Distributor_DistributionType needs to be unique
```
Comment (marshsif) [inline x/dispensation/Readme-Distrubute.md:30]: The Distribution name is BlockHeight_Distributor, therefore, the combination BlockHeight_Distributor_DistributionType needs to be unique

File: x/dispensation/Steps-AuthorizedRunner.md [hunk 5]
```diff
@@ -0,0 +1,200 @@
+# Steps to follow by Authorized Runner
+
+## Requirements
+1. A dispensation should already be created : A dispensation should have been created, and the authorized runner address assigned to it.
```
Comment (marshsif) [inline x/dispensation/Steps-AuthorizedRunner.md:4]: 1. A dispensation should have been created, and the authorized runner address assigned to it.

File: x/dispensation/Steps-AuthorizedRunner.md [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+2. Enough funds to pay for gas fees : The authorized runner would need to pay for the gas fee everytime they run a new transaction.
```
Comment (marshsif) [inline x/dispensation/Steps-AuthorizedRunner.md:5]: 2. The authorized runner would need to pay for the gas fee every time they run a new transaction.

File: x/dispensation/Steps-Claimer.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
+# Steps to follow by User trying to claim rewards
+
+## Requirements
+- The user should have rewards to claim.
+- A user is allowed to claim rewards of type LiquidityMining or ValidatorSubsidy
+- A user can only claim once per distribution cycle , for a particular type. This means a user can have a maximum of two claims (one for ValidatorSubsidy and one for LiquidityMining at any given time)
```
Comment (marshsif) [inline x/dispensation/Steps-Claimer.md:6]: cycle, for

File: x/dispensation/Steps-Claimer.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+## Steps
+### Create claim
+Claims can be created using the CLI or REST interface 
+The usual flow for creating claims would be through the wallet ,which uses the following rest API
```
Comment (marshsif) [inline x/dispensation/Steps-Claimer.md:11]: wallet, which

File: x/dispensation/Steps-Investor.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
+# Steps to follow by Investor/Distributor
```
Comment (marshsif) [inline x/dispensation/Steps-Investor.md:1]: in general just tidy up the punctuation below "1. " and commas, <-space after, not before

File: x/dispensation/Steps-Investor.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+## Requirements
+1 . List of address to distribute to :  For ValidatorSubsidy and LiquidityMining this list will be provided by sifchain ,based on the users who have claimed rewards during the whole week .
+
+2 . Enough funds to distribute : The account balance for the distributor needs to greater than the sum of all the outputs from step 1
+
+3 . Authorized runner address : After creation ,the authorized runner would be responsible for triggering the reward transfers(10 per block)
+
+## Steps
+### Create Dispensation
+Use the CLI to create a dispensations
```
Comment (marshsif) [inline x/dispensation/Steps-Investor.md:12]: remove 's' at end

**Pull request merged**